### PR TITLE
Use language to determine LayoutDirection

### LINKED ISSUE #681: Layout direction does not match language
Hello There, Future Issue Creator! 

Found a bug? Just a friendly heads up, _debugging it requires information from you!_ Make sure the template below is filled out in its entirety. 

 - [x] OS: Windows 7
 - [x] Python Version: CPython 3.7
 - [x] Gooey Version: Gooey 1.0.8. This happens with wxPython 4.0.7 and newer.
 - [x] Thorough description of problem: See below
 - [x] A minimal code example: See below, but note that it requires to set the default language in the OS to a right-to-left language
 - [x] Screenshot (if visual quirk): See below
 - [x] Anything else you may think will be helpful: I will soon provide a PR
 
### Problem Description

When upgrading to a newer wxPython version, suddenly the GUI layout is right-to-left.

Before (Gooey 1.0.2, wxPython 4.0.6):

wxPython-4 0 6

After (Gooey 1.0.2, wxPython 4.0.7):

wxPython-4 0 7

The above is what happens when I don't specify a language. When I specify `language='hebrew'` I get this behavior:

Before (Gooey 1.0.2, wxPython 4.0.6):

wxPython-4 0 6-hebrew

After (Gooey 1.0.2, wxPython 4.0.7):

wxPython-4 0 7-hebrew

In the newer version the layout direction matches the language, but this is a "coincidence": it's only because the language I specified for Gooey is the same as the system default language.

It seems that newer wxPython version give a default LayoutDirection according to the default system language.

### Expected behavior

Since Gooey always uses English as the default language (regardless of the system default), it should override the default LayoutDirection.

Even better, Gooey should just use a LayoutDirection that matches the language.

### Minimal code example

Note: to see the behavior, you will need to set the default language in the OS to a right-to-left language (e.g. Hebrew).

```
from gooey import Gooey, GooeyParser


@Gooey(program_name="gooey_example") #, language="hebrew")
def main():
    parser = GooeyParser(
        description="Simple example"
    )
    parser.add_argument("input_file", type=str, widget="FileChooser", help="Input file")
    parser.add_argument("-v", "--verbose", action="store_true", help="Verbose output")
    args = parser.parse_args()


if __name__ == "__main__":
    main()
```

Thanks! ^_^

### PR DESCRIPTION
**Pull request by tomerv**
Fixes #681.

Hello there! 

Make sure you've followed the [Contributing](https://github.com/chriskiehl/Gooey/blob/master/CONTRIBUTING.md) guidelines before finalizing your pull request. 

TL;DR: 

 - [x] You're opening this PR against the current [release branch](https://github.com/chriskiehl/Gooey/blob/master/CONTRIBUTING.md#development-overview)
 - [x] Works on both Python 2.7 & Python 3.x
 - [x] Commits have been squashed and includes the relevant issue number
 - [x] Pull request description contains link to relevant issue or detailed notes on changes
  - [x] This **must** include example code demonstrating your feature or the bug being fixed
 - [x] All existing tests pass
   - I get "5 failed, 65 passed, 2 warnings, 1 error", but this is the same as what I get when running on `1.0.9-release`
 - [ ] Your bug fix / feature has associated test coverage
   - This one is a bit difficult for me, I couldn't find any existing test to build upon. Seems like the support for languages is missing test coverage?
 - [ ] README.md is updated (if relevant)
   - I think it's not important, but let me know if you think otherwise, and I'll update accordingly.

### CODE REVIEW

**Comment by tomerv** [comment]
@chriskiehl - any chance you can take a look at this PR? Thanks!

**Comment by chriskiehl** [comment]
In time. Tight release schedule at the day job is preventing all open source work atm.

**Comment by ZvikaZ** [comment]
Gooey seems great; but the issue addressed by this PR is making it unusable for people with Hebrew/Arabic/Persian OSes.

It'd be great if you can review this PR.
